hardware_interface/src/resource_manager.cpp

```diff
@@ -1389,7 +1465,7 @@ LoanedCommandInterface ResourceManager::claim_command_interface(const std::strin
   resource_storage_->claimed_command_interface_map_[key] = true;
   std::lock_guard<std::recursive_mutex> guard(resource_interfaces_lock_);
   return LoanedCommandInterface(
-    resource_storage_->command_interface_map_.at(key),
+    *(resource_storage_->command_interface_map_.at(key)),
```

Comment:
why do we need here to dereference this? Don't we provide also a SharedPtr to the Loaned interfaces, as they manage the access?